client/src/main/java/com/kodypay/api/model/RefundResponse.java

```diff
@@ -13,8 +13,6 @@ public class RefundResponse {
     private String orderId;
     @JsonProperty("failureReason")
     private String failureReason;
-    @JsonProperty("receiptJson")
-    private Map<String, Object> receiptJson;
```

Comment:
Please also update the `README` for this and you can set the version to `0.0.5` in the README in anticipation of the new release